.travis.yml

```diff
@@ -58,7 +58,10 @@ jobs:
             - android-29
             - extra-google-m2repository
             - extra-android-m2repository
+      dist: xenial
```

Comment:
I believe we should try to move all builds to a newer distro. Could you please move this to root?

Reply:
Sure.